packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.Props.ts

```diff
@@ -7,11 +7,17 @@ export interface IChoiceGroupProps extends React.HTMLProps<HTMLElement> {
   options: IChoiceGroupOption[];
 
   /**
-   * A callback for receiving a notification when the choice has been changed.
+   * @deprecated
+   * Deprecated and will be removed by v 1.0. Use 'onChange' instead.
```

Comment:
We'll have to stop referring to v1.0 as the release where we remove deprecated props, as our next release is v1.0. Could be just "Deprecated and will be removed in a future release".